### DISCUSSION TITLE
ValueError: File not found

### DISCUSSION THREAD

**Discussion by SimplAiDev** [Linaqruf/kohya-trainer#218] [General]
Hi,

i am trying to train LORA on google colab and when i execute cell 5.5 Start Training i am having this error.

Loading settings from /content/LoRA/config/config_file.toml...
/content/LoRA/config/config_file
prepare tokenizer
update token length: 225
Load dataset config from /content/LoRA/config/dataset_config.toml
╭───────────────────── Traceback (most recent call last) ──────────────────────╮
│ /content/kohya-trainer/train_network.py:752 in <module>                      │
│                                                                              │
│   749 │   args = parser.parse_args()                                         │
│   750 │   args = train_util.read_config_from_file(args, parser)              │
│   751 │                                                                      │
│ ❱ 752 │   train(args)                                                        │
│   753                                                                        │
│                                                                              │
│ /content/kohya-trainer/train_network.py:85 in train                          │
│                                                                              │
│    82 │   blueprint_generator = BlueprintGenerator(ConfigSanitizer(True, Tru │
│    83 │   if use_user_config:                                                │
│    84 │   │   print(f"Load dataset config from {args.dataset_config}")       │
│ ❱  85 │   │   user_config = config_util.load_user_config(args.dataset_config │
│    86 │   │   ignored = ["train_data_dir", "reg_data_dir", "in_json"]        │
│    87 │   │   if any(getattr(args, attr) is not None for attr in ignored):   │
│    88 │   │   │   print(                                                     │
│                                                                              │
│ /content/kohya-trainer/library/config_util.py:485 in load_user_config        │
│                                                                              │
│   482 def load_user_config(file: str) -> dict:                               │
│   483   file: Path = Path(file)                                              │
│   484   if not file.is_file():                                               │
│ ❱ 485 │   raise ValueError(f"file not found / ファイルが見つかりません: {fil │
│   486                                                                        │
│   487   if file.name.lower().endswith('.json'):                              │
│   488 │   try:                                                               │
╰──────────────────────────────────────────────────────────────────────────────╯
ValueError: file not found / ファイルが見つかりません: 
/content/LoRA/config/dataset_config.toml
╭───────────────────── Traceback (most recent call last) ──────────────────────╮
│ /usr/local/bin/accelerate:8 in <module>                                      │
│                                                                              │
│   5 from accelerate.commands.accelerate_cli import main                      │
│   6 if __name__ == '__main__':                                               │
│   7 │   sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])     │
│ ❱ 8 │   sys.exit(main())                                                     │
│   9                                                                          │
│                                                                              │
│ /usr/local/lib/python3.10/dist-packages/accelerate/commands/accelerate_cli.p │
│ y:45 in main                                                                 │
│                                                                              │
│   42 │   │   exit(1)                                                         │
│   43 │                                                                       │
│   44 │   # Run                                                               │
│ ❱ 45 │   args.func(args)                                                     │
│   46                                                                         │
│   47                                                                         │
│   48 if __name__ == "__main__":                                              │
│                                                                              │
│ /usr/local/lib/python3.10/dist-packages/accelerate/commands/launch.py:1104   │
│ in launch_command                                                            │
│                                                                              │
│   1101 │   elif defaults is not None and defaults.compute_environment == Com │
│   1102 │   │   sagemaker_launcher(defaults, args)                            │
│   1103 │   else:                                                             │
│ ❱ 1104 │   │   simple_launcher(args)                                         │
│   1105                                                                       │
│   1106                                                                       │
│   1107 def main():                                                           │
│                                                                              │
│ /usr/local/lib/python3.10/dist-packages/accelerate/commands/launch.py:567 in │
│ simple_launcher                                                              │
│                                                                              │
│    564 │   process = subprocess.Popen(cmd, env=current_env)                  │
│    565 │   process.wait()                                                    │
│    566 │   if process.returncode != 0:                                       │
│ ❱  567 │   │   raise subprocess.CalledProcessError(returncode=process.return │
│    568                                                                       │
│    569                                                                       │
│    570 def multi_gpu_launcher(args):                                         │
╰──────────────────────────────────────────────────────────────────────────────╯
CalledProcessError: Command '['/usr/bin/python3', 'train_network.py', 
'--sample_prompts=/content/LoRA/config/sample_prompt.txt', 
'--dataset_config=/content/LoRA/config/dataset_config.toml', 
'--config_file=/content/LoRA/config/config_file.toml']' returned non-zero exit 
status 1.

Thank You for your Help

**Comment by Linaqruf** [comment]
Hi, you must be forgot to set run `5.2. dataset config`

**Reply by yzufardiansyah** [reply]
thank you, its work
